xformers/components/attention/nystrom.py

```diff
@@ -57,24 +57,28 @@ class NystromSelfAttentionConfig(AttentionConfig):
     use_razavi_pinverse: Optional[bool]
 
 
-def get_avg_pool(n: int):
-    def avg_pool(x: torch.Tensor):
+class AvgPool:
```

Comment:
I think that mypy is right, this is missing the nn.Module inheritance probably

`class AvgPool(nn.Module):`
should be enough to fix ?